client/landing/jetpack-cloud/components/server-connection-indicator/placeholder.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ServerConnectionIndicatorPlaceholder = () => {
+	const translate = useTranslate();
+
+	return (
+		<Card className="server-connection-indicator__placeholder">
+			<div className="server-connection-indicator__placeholder-body">
+				<svg
+					viewBox="0 0 100 100"
+					xmlns="http://www.w3.org/2000/svg"
+					className="server-connection-indicator__placeholder-image "
+				>
+					<circle cx="50" cy="50" r="50" />
+				</svg>
+
+				<div className="server-connection-indicator__placeholder-info">
+					<h4 className="server-connection-indicator__placeholder-status">
+						{ translate( 'Server Status:' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'Server Status' )` *ES Score: 10*